README.md

```diff
@@ -91,11 +91,12 @@ You can check out the GitHub Explore website [at github.com/explore](https://git
 
 ## 📝 Next steps 
 
-* Open a pull request and let your teacher know that you’ve finished this course.  
+* Open a pull request and let your instructor know that you’ve finished this course.  
 * Create a new markdown file in this repository. Let them know what you learned and what you are still confused about! Experiment with different styles!
```

Comment:
### Markdown File Nelson Meyn
I'm still a little confused as to what **markdown** files are, but I think I have figured it out. _I have learned the basics of GitHub, while scary I think I got used to it._